container/broker/kvm-broker_test.go

```diff
@@ -328,155 +322,3 @@ func (s *kvmBrokerSuite) TestStartInstanceWithContainerInheritProperties(c *gc.C
 		},
 	}, c)
 }
-
```

Comment:
The kvmProvisionerSuite covers tests for the container provisioner using a kvm container broker.  Moved to a new file in worker/provisioner.